src/database/standalone-sqlite.test.ts

```
@@ -1506,6 +1506,107 @@ describe('StandaloneSqliteDatabase', () => {
+        'root_transaction_id must survive optimistic re-POST',
+      );
+      assert.notEqual(
+        row!.data_offset,
+        null,
+        'data_offset must survive optimistic re-POST',
+      );
+      assert.deepEqual(row!.parent_id, fromB64Url(bundleParentId));
+      assert.deepEqual(
+        row!.root_transaction_id,
+        fromB64Url(bundleRootTxId),
+      );
```

ESLint

[error] Replace `⏎········row!.root_transaction_id,⏎········fromB64Url(bundleRootTxId),⏎······` with `row!.root_transaction_id,·fromB64Url(bundleRootTxId)`

(prettier/prettier)